plugin/backend_linux.go

```diff
@@ -228,10 +234,45 @@ func (pm *Manager) Pull(name string, metaHeader http.Header, authConfig *types.A
 }
 
 // List displays the list of plugins and associated metadata.
-func (pm *Manager) List() ([]types.Plugin, error) {
+func (pm *Manager) List(filter string) ([]types.Plugin, error) {
```

Comment:
I think we should pass here a `filters.Args` instead of a string that is a json one. This is what we do on other filters (like on containers, …)